independent-projects/resteasy-reactive/pom.xml

```diff
@@ -46,7 +46,6 @@
         <!-- Versions -->
         <jakarta.enterprise.cdi-api.version>4.1.0</jakarta.enterprise.cdi-api.version>
         <jandex.version>3.2.3</jandex.version>
-        <bytebuddy.version>1.14.11</bytebuddy.version>
```

Comment:
It shouldn't use this old version but these projects are supposed to be independent so it was expected to have this version here.

Reply:
Oh, `quarkus-parent` is not the top-level pom. I'll fix my commit then.

Reply:
And done.